data/core/units/monsters/Cat_Jump.cfg

```diff
@@ -0,0 +1,134 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Jump Cat
+    name= _ "Jump Cat"
+    race=cats
+    generate_name=no
+    image="units/monsters/cat/jumpcat.png"
+    profile="portraits/monsters/jumpcat.webp"
+    hitpoints=32
+    movement=9
+    # similar to elves, but not as bad in caves, and not magical
+    movement_type=woodland
+    [defense]
+        cave=60
+        frozen=60
+    [/defense]
+    [movement_costs]
+        cave=2
+        mountains=2
+    [/movement_costs]
+    [resistance]
+        arcane=80
+    [/resistance]
+    experience=100
+    {AMLA_DEFAULT}
+    level=1
+    alignment=chaotic
+    advances_to=null
+    cost=15
+    usage=scout
+    description=_ "Jumpcats are opportunistic predators with keen senses and fast reflexes, but very little bulk, so they are easily dispatched if they can be hit, and easily scared off if they can't.  One of the most distinctive features of these odd creatures is the hard ball they carry on their tail, which is their primary offensive weapon.  Their actions are mostly driven by instinct, so they are sometimes used as assassins by more sophisticated handlers."
+    # To Do: Get smaller cat sounds  
+    die_sound=drake-die.ogg
+    [abilities]
+        {ABILITY_AMBUSH}
+    [/abilities]
+    [standing_anim]
+        start_time=-50
+        [frame]
+            image="units/monsters/cat/jumpcat.png:150"
+        [/frame]
+    [/standing_anim] 
+    [standing_anim]
+        start_time=-50
+        {STANDARD_IDLE_FILTER}
+        [frame]
+            image="units/monsters/cat/jumpcat.png:150"
+        [/frame]
+    [/standing_anim] 
+#ifdef __UNUSED__
+    [pre_movement_anim]
+        start_time=0
+        {STANDARD_IDLE_FILTER}
+        [frame]
+            image="units/monsters/cat/tritail-transition[2,1].png:100"
```

Comment:
Copy-paste error?

Reply:
No, it's commented out because the images don't exist yet.  I'll adjust things as needed when there is something to work with, but the tritail stuff is a good starting point.